src/qt/bitcoingui.cpp

```diff
@@ -1034,37 +1035,37 @@ void BitcoinGUI::message(const QString &title, const QString &message, unsigned
     // Prefer supplied title over style based title
     if (!title.isEmpty()) {
         msgType = title;
-    }
-    else {
+    } else {
         switch (style) {
         case CClientUIInterface::MSG_ERROR:
-            msgType = tr("Error");
+            msgType = tr("ERROR");
+            message = tr("ERROR: %1").arg(message);
```

Comment:
Is there a reason this is adding an "ERROR: " prefix for errors generically but using an "Error: " prefix for the disk space error? Why not be consistent?

Reply:
Fixed.